dist/index.html

```diff
@@ -10,50 +10,52 @@
 <body>
   <section class="header">
     <img src="./images/around-the-world.png" class="globe" alt="spinning globe">
-    <div class="welcome">Globetrotter</div>
+    <div tabindex="0"class="welcome">Globetrotter</div>
     <img src="./images/around-the-world.png" class="globe" alt="spinning globe">
   </section> 
   <hr class="line">
-  <div class="welcome-user">Welcome</div>
```

Comment:
if you have time later, maybe consider using articles, sections, or something else rather than a `div.` Most elements have built in properties so you don't need so many tabIndex's . <-- These act as a tabbing bandaid from what I understand.